src/validation.cpp

```diff
@@ -805,8 +803,7 @@ bool MemPoolAccept::PreChecks(ATMPArgs& args, Workspace& ws)
     // If we don't hold the lock allConflicting might be incomplete; the
     // subsequent RemoveStaged() and addUnchecked() calls don't guarantee
     // mempool consistency for us.
-    fReplacementTransaction = setConflicts.size();
-    if (fReplacementTransaction)
+    if (setConflicts.size())
     {
```

Comment:
nit: braces go on the same line

```suggestion
    if (setConflicts.size()) {
```

Reply:
Done.